pkg/inject/sidecar.go

```diff
@@ -274,6 +275,26 @@ func container(jaeger *v1.Jaeger, dep *appsv1.Deployment) corev1.Container {
 				Name:          "admin-http",
 			},
 		},
+		LivenessProbe: &corev1.Probe{
+			Handler: corev1.Handler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Path: "/",
+					Port: intstr.FromInt(int(adminPort)),
```

Comment:
What happens if the other container, with the business application, exposes the same port?

Reply:
Hi @jpkrohling that's a very good point, is there already any logic to verify that the other ports do not clash either?

Reply:
I don't think we have it, as this is the only place where we deal with resources we don't create ourselves. Shouldn't be hard, though: I think we can just skip adding the probes in case the main container has the port being declared?

Reply:
@jpkrohling I've added a function to check if the port is available.